content/_changelogs/11.2.0.md

```diff
@@ -0,0 +1,26 @@
+## 11.2.0
+
+_Released 11/22/2022_
+
+**Features:**
+
+- Re-implemented the 'Run All Specs' for End to End Testing for running multiple tests sequentially in open mode. This feature had been previously removed in Cypress 10.0. Enable 'Run All Specs' with the new experimental flag, `experimentalRunAllSpecs`. Please leave any feedback around 'Run All Specs'
+  [here](https://github.com/cypress-io/cypress/discussions/21628). Your feedback
+  will help us make product decisions around the future of this feature. Addresses
```

Comment:
oops needs fixed with merged suggestion

Reply:
Yeah this was a bad merge. Will be fixed in a sec.